index.html

```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
            <img id="img" src="./features/images/Disabled.png">
       </section>
      </main>
+     <section>
+        <div id="content2"><p class="works">My Recent Works</p>
```

Comment:
- [ ] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.